roles/repositories/README.md

```diff
@@ -0,0 +1,131 @@
+    repository_sets:
+      - name: Red Hat Software Collections RPMs for Red Hat Enterprise Linux 7 Server
+        basearch: x86_64
+        releasever: 7Server
+      - name: Red Hat Software Collections RPMs for Red Hat Enterprise Linux 6 Server
+        basearch: x86_64
+        releasever: 6Server
+  - name: Red Hat Enterprise Linux for x86_64
+    repository_sets:
+      - name: Red Hat Enterprise Linux 8 for x86_64 - BaseOS (RPMs)
+        releasever: 8
+      - name: Red Hat Enterprise Linux 8 for x86_64 - AppStream (RPMs)
+        releasever: 8
+  - name: CentOS 8
+    repositories:
+      - name: BaseOS x86_64
+        content_type: yum
+        url: http://mirror.centos.org/centos/8/BaseOS/x86_64/os/
+  - name: Debian 10
+    repositories:
+      - name: main
+        content_type: deb
+        url: http://deb.debian.org/debian
+        deb_components: main
+        deb_architectures: amd64
+        deb_releases: buster
+  - name: Foreman Client
+    repositories:
+      - name: Foreman Client Debian
+        url: http://deb.atix.de/debian
+        content_type: deb
+        deb_components: main
+        deb_architectures: amd64
+        deb_releases: buster
+      - name: Foreman Client CentOS
+        url: http://yum.theforeman.org/client/latest
+        content_type: yum
+```
+
+- `sync_plans`: List of sync plans to create. Each sync plan is represented as a dictionary which specifies the `name` of the sync plan and the `products` assigned to the sync plan. It also specifies the `interval` which can be 'hourly', 'daily', 'weekly', or 'custom cron'. In case the 'custom cron' `interval` is used, it should also specify the `cron_expression`. Finally the sync plan should have a `sync_date` which specifies the first time that the sync plan will run.
+
+```yaml
+sync_plans:
+  - name: Weekly Sync
+    interval: weekly
+    sync_date: 2020-11-07 00:00:00 UTC
+    products:
+      - Red Hat Enterprise Linux Server
+      - Red Hat Software Collectiions (for RHEL Server)
+      - Red Hat Enterprise Linux for x86_64
+      - CentOS 8
+      - Debian 10
+  - name: Monthly Foreman Client Sync
+    interval: custom cron
+    cron_expression: 0 6 8 * *
+    sync_date: 2020-11-08 00:06:00 UTC
+    products:
+      - Foreman Client
+```
+
+Example Playbooks
+-----------------
+
+Enable several Red Hat Repositories and create a weekly sync plan. There are a few important points to note about the structure of the data in this example. The first is that RHEL 8 repos have a different product name than previous RHEL versions. Additionally the RHEL 8 product already contains the `basearch` so it should not be specified on the RHEL 8 `repository_sets`, and the naming convention for `releasever` changed with RHEL 8 since system purpose removes the need for separate distributions like `Server` and `Workstation`. Finally, the optional and extras repositories do not have point releases so `releasever` should be omitted.
```

Comment:
Maybe make the "points" a bullet list.

Reply:
I've made a first pass at cleaning that up